include/rocksdb/db.h

```diff
@@ -602,6 +602,24 @@ class DB {
     return Status::NotSupported("GetEntity not supported");
   }
 
+  // Returns logically grouped wide-column entities per column family (a.k.a.
+  // attribute groups) for a single key. PinnableAttributeGroups is a vector of
+  // PinnableAttributeGroup. Each PinnableAttributeGroup will have
+  // ColumnFamilyHandle*, Status and PinnableWideColumns as output.
```

Comment:
`ColumnFamilyHandle*` as input, right?

Reply:
`ColumnFamilyHandle*`should be already part of each `PinnableAttributeGroup` that is required to be passed in.